src/macrofier/engine.mjs

```diff
@@ -1129,7 +1132,7 @@ function insertMethodMacros(template, methodObj, json, templates, examples = {})
     .replace(/\$\{event\.callback\.params\.serialization\}/g, callbackSerializedParams)
     .replace(/\$\{event\.callback\.result\.instantiation\}/g, callbackResultInst)
     .replace(/\$\{event\.callback\.response\.instantiation\}/g, callbackResponseInst)
-    .replace(/\$\{info\.title\}/g, info.title)
```

Comment:
This will break several things. Why are you modifying `info.title` to be lowercase?

Reply:
Reverted this back. this was added to get module name in jsonrpc request in lower case. Since the ripple support both case, reverted it back